frontend/web/routes.js

```diff
@@ -17,7 +17,7 @@ import AccountSettingsPage from './components/pages/AccountSettingsPage'
 import NotFoundErrorPage from './components/pages/NotFoundErrorPage'
 import ProjectSettingsPage from './components/pages/ProjectSettingsPage'
 import PasswordResetPage from './components/pages/PasswordResetPage'
-import EnvironmentSettingsPage from './components/pages/EnvironmentSettingsPage'
+import EnvironmentSettingsPage from './components/pages/EnvironmentSettingsPage.tsx'
```

Comment:
Can you remove `EnvironmentSettingsPage.js` file and change this? 
```suggestion
import EnvironmentSettingsPage from './components/pages/EnvironmentSettingsPage'
```